### DISCUSSION TITLE
OpenCV Headless version

### DISCUSSION THREAD

**Discussion by milosacimovic** [felixdittrich92/OnnxTR#34] [Ideas]
Hi Felix, 

I was thinking that as OnnxTR is a deployment package it would make more sense to have opencv-python-headless specified in its requirements as it is a more lightweight version.

What do you think?

Kindly,
Milos

**Comment by felixdittrich92** [comment]
Hi @milosacimovic 👋🏼,

Yeah that's a good idea.

I think two additional install options makes sense `cpu-headless` and `gpu-headless` in this case we keep the other both `cpu` and `gpu` as is because most other packages relies on the non headless version which would be conflicting.

Wdyt ? :)

**Comment by felixdittrich92** [comment]
@milosacimovic Opened a PR to add it :)

I would say if we are done with your PR in docTR and it's merged i can sync it to OnnxTR and do a release :+1: 
Until this i will keep the headless PR - to avoid confiusion with the Readme :sweat_smile:

**Reply by milosacimovic** [reply]
Sounds good, I'll finish up the docTr PR first. I should have it by the end of this week. Thanks so much for the reply!

**Reply by felixdittrich92** [reply]
How does it look @milosacimovic  ? ☺️

**Reply by milosacimovic** [reply]
Hi @felixdittrich92 I've just put in your changes to a new pull request in docTr repo and I've slightly modified the extract_rcrops which works a bit better according to my internal tests.

**Comment by felixdittrich92** [comment]
@milosacimovic done :hugs: 
--> https://github.com/felixdittrich92/OnnxTR/releases/tag/v0.5.0  / https://pypi.org/project/onnxtr/

**Reply by milosacimovic** [reply]
Amazing, thanks @felixdittrich92 !!!
